docs/orchestrating-elastic-stack-applications/agent.asciidoc

```diff
@@ -0,0 +1,403 @@
+
+NOTE: Please note that link:https://www.elastic.co/guide/en/fleet/current/elastic-agent-installation.html[enrollment in Fleet] is currently not supported.
+
+
+
+* <<{p}-elastic-agent-quickstart,Quickstart>>
+* <<{p}-elastic-agent-configuration,Configuration>>
+* <<{p}-elastic-agent-configuration-examples,Configuration Examples>>
+
+[id="{p}-elastic-agent-quickstart"]
+== Quickstart
+
+experimental[]
+
+. Apply the following specification to deploy Elastic Agent with the System metrics integration to harvest CPU metrics from the Agent Pods. ECK automatically configures the secured connection to an Elasticsearch cluster named `quickstart`, created in the link:k8s-quickstart.html[Elasticsearch quickstart].
++
+[source,yaml,subs="attributes,+macros"]
+----
+cat $$<<$$EOF | kubectl apply -f -
+apiVersion: agent.k8s.elastic.co/v1alpha1
+kind: Agent
+metadata:
+  name: quickstart
+spec:
+  version: {version}
+  elasticsearchRefs:
+  - name: quickstart
+  daemonSet: {}
+  config:
+    inputs:
+      - name: system-1
+        revision: 1
+        type: system/metrics
+        use_output: default
+        meta:
+          package:
+            name: system
+            version: 0.9.1
+        data_stream:
+          namespace: default
+        streams:
+          - id: system/metrics-system.cpu
+            data_stream:
+              dataset: system.cpu
+              type: metrics
+            metricsets:
+              - cpu
+            cpu.metrics:
+              - percentages
+              - normalized_percentages
+            period: 10s
+EOF
+----
++
+See <<{p}-elastic-agent-configuration-examples>> for more ready-to-use manifests.
+
+. Monitor Elastic Agent.
++
+Retrieve the status of Elastic Agent.
++
+[source,sh]
+----
+kubectl get agent
+----
++
+[source,sh,subs="attributes"]
+----
+NAME            HEALTH   AVAILABLE   EXPECTED   VERSION   AGE
+quickstart      green    3           3          {version}    15s
+
+----
+
+. List all the Pods belonging to a given Elastic Agent specification.
++
+[source,sh]
+----
+kubectl get pods --selector='agent.k8s.elastic.co/name=quickstart'
+----
++
+[source,sh]
+----
+NAME                     READY   STATUS    RESTARTS   AGE
+quickstart-agent-6bcxr   1/1     Running   0          68s
+quickstart-agent-t49fd   1/1     Running   0          68s
+quickstart-agent-zqp55   1/1     Running   0          68s
+----
+
+. Access logs for one of the Pods.
++
+[source,sh]
+----
+kubectl logs -f quickstart-agent-6bcxr
+----
+
+. Access the CPU metrics ingested by Elastic Agent.
++
+You have two options:
++
+- Follow the Elasticsearch deployment <<{p}-deploy-elasticsearch,guide>> and run:
++
+[source,sh]
+----
+curl -u "elastic:$PASSWORD" -k "https://localhost:9200/metrics-system.cpu-*/_search"
+----
++
+- Follow the Kibana deployment <<{p}-deploy-kibana,guide>>, log in and go to *Kibana* > *Discover*.
+
+[id="{p}-elastic-agent-configuration"]
+== Configuration
```

Comment:
Maybe we should also add a paragraph on how to install dashboards etc via the Kibana Fleet UI